src/formatter/index.ts

```diff
@@ -176,35 +176,25 @@ export class QueryFormatter {
     // YYYY-MM-DD HH:mm:ss.mmm
     const yearMonthDay =
       zeroPad(year, 4) + "-" + zeroPad(month, 2) + "-" + zeroPad(day, 2);
-    const hourMinuteSecond =
-      zeroPad(hour, 2) + ":" + zeroPad(minute, 2) + ":" + zeroPad(second, 2);
+    const time =
+      zeroPad(hour, 2) +
+      ":" +
+      zeroPad(minute, 2) +
+      ":" +
+      zeroPad(second, 2) +
+      "." +
+      zeroPad(millisecond, 3);
 
     if (param instanceof PGDate) {
       return this.escapeString(yearMonthDay);
     }
 
     if (param instanceof TimestampTZ) {
-      const str =
-        yearMonthDay +
-        " " +
-        hourMinuteSecond +
-        "." +
-        zeroPad(millisecond, 6, "right") +
-        " " +
-        param.timeZone;
+      const str = yearMonthDay + " " + time + " " + param.timeZone;
       return this.escapeString(str);
     }
 
-    if (param instanceof TimestampNTZ) {
-      const str =
-        yearMonthDay +
-        " " +
-        hourMinuteSecond +
-        "." +
-        zeroPad(millisecond, 6, "right");
-      return this.escapeString(str);
-    }
-    const str = yearMonthDay + " " + hourMinuteSecond;
```

Comment:
before: here we were serializing without time, wich was a default for `Date` type
after: str is with time